libfwupdplugin/fu-context.c

```diff
@@ -1001,6 +1001,61 @@ fu_context_housekeeping(FuContext *self)
 
 typedef gboolean (*FuContextHwidsSetupFunc)(FuContext *self, FuHwids *hwids, GError **error);
 
+/**
+ * fu_context_get_fde:
+ * @self: a #FuContext
+ * @flags: a #FuContextFlags
+ *
+ * Determine whether a given type of FDE as passed to the function is
+ * enabled.
+ *
+ * Returns: gboolean
+ *
+ * Since: 2.05
+ **/
+gboolean
+fu_context_get_fde(FuContext *self, FuContextFlags flags)
+{
+	FuContextPrivate *priv = GET_PRIVATE(self);
+	g_return_val_if_fail(FU_IS_CONTEXT(self), FALSE);
+	return priv->flags & flags;
+}
+
+/**
+ * fu_context_detect_full_disk_encryption:
+ * @self: a #FuContext
+ *
+ * Detects if full disk encryption is enabled.
+ *
+ * Since: 2.0.5
+ **/
```

Comment:
if it's static you don't need the comment block

Reply:
dropped